client/debian/rules

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/make -f
+
+# Defines DEB_*_RUST_TYPE triples
+include /usr/share/rustc/architecture.mk
+
+export CARGO_HOME = $(CURDIR)/debian/
+export CARGO_VENDOR_DIR = $(CURDIR)/rust-vendor/
+# Enable debug symbols to avoid debug-file-with-no-debug-symbols warning
+export CARGO_PROFILE_RELEASE_DEBUG=1
+
+CARGO_TARGET_DIR = $(CURDIR)/target
+PARALLEL := $(shell nproc)
```

Comment:
I believe there is an environment variable for exactly this purpose, but I don't recall its name.